containers/anndata-to-ui/context/main.py

```diff
@@ -0,0 +1,68 @@
+import argparse
+from glob import glob
+from pathlib import Path
+from os import mkdir, environ
+import json
+
+import zarr
+from scipy import sparse
+from anndata import read_h5ad
+
+NUM_MARKER_GENES_TO_VISUALIZE = 5
+VAR_CHUNK_SIZE = 10
+
+
+def main(input_dir, output_dir):
+    output_dir.mkdir(exist_ok=True)
+    for h5ad_file in ["secondary_analysis.h5ad", "scvelo_annotated.h5ad"]:
+        adata = read_h5ad(input_dir / h5ad_file)
+        if "rank_genes_groups" in adata.uns:
+            # Handle marker genes by putting top n per cluster in `obs` for `factors` visualization.
+            marker_genes = []
+            for i in range(NUM_MARKER_GENES_TO_VISUALIZE):
+                adata.obs[f"marker_gene_{str(i)}"] = ["" for v in adata.obs.index]
+                for cluster in adata.obs["leiden"]:
+                    marker_gene = adata.uns["rank_genes_groups"]["names"][i][cluster]
+                    adata.obs[f"marker_gene_{str(i)}"][
+                        adata.obs["leiden"] == cluster
+                    ] = marker_gene
+                    marker_genes.append(marker_gene)
+            adata.var["marker_genes_for_heatmap"] = [
+                gene in marker_genes for gene in adata.var.index
+            ]
+        if "dispersions_norm" in adata.var:
+            top_50_dispersion = adata.var["dispersions_norm"][
```

Comment:
Marker genes look better than the top `n` highly variable for the heatmap but just in case that doesn't pan out in general, or for some reason RNA-seq and slide-seq don't align in that respect, we will do this.  We'll support forming new gene sets anyway in Vitessce soon (i.e you will be able to create a gene set from the top `n` dispersion norms to view in a heatmap on the fly) but for now, just want this as a fallback.